tools/ci_build/github/azure-pipelines/orttraining-1p-linux-gpu-ci-pipeline.yml

```diff
@@ -0,0 +1,69 @@
+parameters:
+  - name: build_flag
+    displayName: Build Flag
+    type: string
+    default: --enable_onnx_tests
+  - name: image_tag
+    displayName: Image Tag
+    type: string
+    default: latest
+
+stages:
+- stage: Python_Packaging
+
+  variables:
+    docker_image_prefix: onnxruntime-training
+    linux_gpu_dockerfile: ../../../../../dockerfiles/Dockerfile.training
+    python.version: 3.6
+    BuildCommand: >
+      tools/ci_build/github/linux/run_dockerbuild.sh
+      -o ubuntu16.04 -d gpu -r $(Build.BinariesDirectory)
+      -x "
+      --enable_training
+      --config $(buildConfig)
+      --skip_onnx_tests
+      --build_wheel
+      --enable_nvtx_profile
+      "
+  jobs:
+  - job: Linux_py_GPU_Wheels
+    timeoutInMinutes: 0
+    workspace:
+      clean: all
+    pool: Linux-GPU-CUDA10
+    #pool: Linux-Multi-GPU-V100
+    steps:
+      - checkout: self
+        clean: true
+        submodules: recursive
+
+      - template: templates/set-py-packaging-variables-step.yml
+
+      - task: CmdLine@2
+        inputs:
+          script: |
+            docker build \
+              --pull \
+              -t ${{ variables.docker_image_prefix }}-manylinux-gpu-release \
+              --build-arg BUILD_FLAG="${{ parameters.build_flag }}" \
+              -f ${{ variables.linux_gpu_dockerfile }} .
+          workingDirectory: $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker
+
+      - task: AzureKeyVault@1
+        inputs:
+          azureSubscription: 'AIInfraBuild'
+          KeyVaultName: 'jingywatest'
+          SecretsFilter: '*'
+          RunAsPreJob: false
+
+      - task: CmdLine@2
+        inputs:
+          script: |
+            echo $(OrtreleasePassword) > password.txt
+            cat password.txt | docker login --username ortrelease --password-stdin ortrelease.azurecr.io
+            docker tag onnxruntime-ubuntu16.04-cuda10.1-cudnn7.6 ortrelease.azurecr.io/onnxruntime-training:${{ parameters.image_tag }}
```

Comment:
the image being tagged here doesn't seem to match the one built earlier

Reply:
Fixed